pymesync.py

```diff
@@ -535,6 +535,10 @@ def __local_auth_error(self):
 
     def __response_to_python(self, response):
         """Convert response to native python list of objects"""
+        # DELETE returns an empty body if successful
+        if not response.text and response.status_code == 200:
```

Comment:
@matthewrsj Hmm, this seems like a bad function name if it's only used for DELETES. If you could change the name or change the function to reflect the name/docstring it'd sit better with me.

``__response_to_python`` feels like a method which always converts a response to a python data-type.